src/validation/ListKnownPartiesRequestValidation.ts

```diff
@@ -1,22 +1,8 @@
 // Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import {TransactionFilterValidation} from "./TransactionFilterValidation";
 import {ListKnownPartiesRequest} from "../model/ListKnownPartiesRequest";
-import {OptionalFieldsValidators, RequiredFieldsValidators} from "./Validation";
-import {native} from "./Native";
+import {noFields} from "./Validation";
 import {object} from "./Object";
 
-function required(): RequiredFieldsValidators<ListKnownPartiesRequest> {
-    return {
-        filter: TransactionFilterValidation,
-    };
-}
-
-function optional(): OptionalFieldsValidators<ListKnownPartiesRequest> {
-    return {
-        verbose: native('boolean'),
-    }
-}
-
-export const ListKnownPartiesRequestValidation = object<ListKnownPartiesRequest>('GetActiveContractsRequest', required, optional);
+export const ListKnownPartiesRequestValidation = object<ListKnownPartiesRequest>('ListKnownPartiesRequest', noFields, noFields);
```

Comment:
Is this the correct way of validating a GRPC call with no body?

Reply:
Have a look at what the `LedgerIdentityClient` does. I suspect there's simply no validation because there's nothing to validate. And no codec either.